packages/web/localizations/es.json

```diff
@@ -461,5 +461,10 @@
   "autonomy": {
     "description": "Explora Órdenes de Límite y Stop presentadas por Autonomy: ",
     "link": "osmosis.autoswap.trade"
-  }
-}
+  },
+  "errors.fallbackText1": "Ocurrió un error",
```

Comment:
Oh interesting. Seems like inlang doesn't know how to embed a child member to the JSON object

Reply:
I reported this FYI